From 4685d84657b579297dd52cf433e2b9a498547330 Mon Sep 17 00:00:00 2001
From: SrujanaChalla <schalla@marvell.com>
Date: Thu, 1 Aug 2019 12:44:29 +0530
Subject: [PATCH 304/767] crypto: restructure marvell directory

commit c007d3fe45392f57289555bc837aa014deb32cd7 from
git@git.assembla.com:cavium/WindRiver.linux.git

We are going to upstream CPT kernel driver(otx, otx2). The respective
files will be placed in drivers/crypto/marvell. But it already has
marvell CESA driver files (namely cesa.c  cesa.h cipher.c  hash.c
Makefile  tdma.c).
So moving the above existing files to a new directory(cesa)
under drivers/crypto/marvell.

Change-Id: Ic8c0a539d34dac44dd02199a9884112ab570b237
Signed-off-by: SrujanaChalla <schalla@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/13448
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
Tested-by: Stefan Chulski <stefanc@marvell.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 MAINTAINERS                                | 2 +-
 drivers/crypto/Makefile                    | 2 +-
 drivers/crypto/marvell/{ => cesa}/Makefile | 0
 drivers/crypto/marvell/{ => cesa}/cesa.c   | 0
 drivers/crypto/marvell/{ => cesa}/cesa.h   | 0
 drivers/crypto/marvell/{ => cesa}/cipher.c | 0
 drivers/crypto/marvell/{ => cesa}/hash.c   | 0
 drivers/crypto/marvell/{ => cesa}/tdma.c   | 0
 8 files changed, 2 insertions(+), 2 deletions(-)
 rename drivers/crypto/marvell/{ => cesa}/Makefile (100%)
 rename drivers/crypto/marvell/{ => cesa}/cesa.c (100%)
 rename drivers/crypto/marvell/{ => cesa}/cesa.h (100%)
 rename drivers/crypto/marvell/{ => cesa}/cipher.c (100%)
 rename drivers/crypto/marvell/{ => cesa}/hash.c (100%)
 rename drivers/crypto/marvell/{ => cesa}/tdma.c (100%)

diff --git a/MAINTAINERS b/MAINTAINERS
index c2a7491ddc68..863ce2ead1db 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -9435,7 +9435,7 @@ F:	Documentation/devicetree/bindings/phy/phy-mvebu-utmi.txt
 MARVELL CRYPTO DRIVER
 M:	Boris Brezillon <bbrezillon@kernel.org>
 M:	Arnaud Ebalard <arno@natisbad.org>
-F:	drivers/crypto/marvell/
+F:	drivers/crypto/marvell/cesa/
 S:	Maintained
 L:	linux-crypto@vger.kernel.org
 
diff --git a/drivers/crypto/Makefile b/drivers/crypto/Makefile
index a23a7197fcd7..a48e7b907c40 100644
--- a/drivers/crypto/Makefile
+++ b/drivers/crypto/Makefile
@@ -15,7 +15,7 @@ obj-$(CONFIG_CRYPTO_DEV_GEODE) += geode-aes.o
 obj-$(CONFIG_CRYPTO_DEV_HIFN_795X) += hifn_795x.o
 obj-$(CONFIG_CRYPTO_DEV_IMGTEC_HASH) += img-hash.o
 obj-$(CONFIG_CRYPTO_DEV_IXP4XX) += ixp4xx_crypto.o
-obj-$(CONFIG_CRYPTO_DEV_MARVELL_CESA) += marvell/
+obj-$(CONFIG_CRYPTO_DEV_MARVELL_CESA) += marvell/cesa/
 obj-$(CONFIG_CRYPTO_DEV_MEDIATEK) += mediatek/
 obj-$(CONFIG_CRYPTO_DEV_MXS_DCP) += mxs-dcp.o
 obj-$(CONFIG_CRYPTO_DEV_NIAGARA2) += n2_crypto.o
diff --git a/drivers/crypto/marvell/Makefile b/drivers/crypto/marvell/cesa/Makefile
similarity index 100%
rename from drivers/crypto/marvell/Makefile
rename to drivers/crypto/marvell/cesa/Makefile
diff --git a/drivers/crypto/marvell/cesa.c b/drivers/crypto/marvell/cesa/cesa.c
similarity index 100%
rename from drivers/crypto/marvell/cesa.c
rename to drivers/crypto/marvell/cesa/cesa.c
diff --git a/drivers/crypto/marvell/cesa.h b/drivers/crypto/marvell/cesa/cesa.h
similarity index 100%
rename from drivers/crypto/marvell/cesa.h
rename to drivers/crypto/marvell/cesa/cesa.h
diff --git a/drivers/crypto/marvell/cipher.c b/drivers/crypto/marvell/cesa/cipher.c
similarity index 100%
rename from drivers/crypto/marvell/cipher.c
rename to drivers/crypto/marvell/cesa/cipher.c
diff --git a/drivers/crypto/marvell/hash.c b/drivers/crypto/marvell/cesa/hash.c
similarity index 100%
rename from drivers/crypto/marvell/hash.c
rename to drivers/crypto/marvell/cesa/hash.c
diff --git a/drivers/crypto/marvell/tdma.c b/drivers/crypto/marvell/cesa/tdma.c
similarity index 100%
rename from drivers/crypto/marvell/tdma.c
rename to drivers/crypto/marvell/cesa/tdma.c
-- 
2.31.1

